org.mixedrealitytoolkit.standardassets/Fonts/Selawik-Bold-MRTKIcons.ttf.meta

```diff
@@ -1,5 +1,5 @@
 fileFormatVersion: 2
-guid: 52767c1f1dd45304da1f49570e5e91ea
+guid: 3fa3a351e0ae62a44948b05c0e4beaea
```

Comment:
Don't change the font GUID's so to not break other's depending on it....

Keep the GUIDs for ALL fonts